src/setup_node_env/node_version_validator.js

```diff
@@ -9,23 +9,25 @@
 
 var pkg = require('../../package.json');
 
-// Note: This is written in ES5 so we can run this before anything else
-// and gives support for older NodeJS versions
-var currentVersion = (process && process.version) || null;
-var rawRequiredVersion = (pkg && pkg.engines && pkg.engines.node) || null;
-var requiredVersion = rawRequiredVersion ? 'v' + rawRequiredVersion : rawRequiredVersion;
-var isVersionValid = !!currentVersion && !!requiredVersion && currentVersion === requiredVersion;
+if (!process.env.UNSAFE_DISABLE_NODE_VERSION_VALIDATION) {
```

Comment:
@elastic/kibana-operations I had to add this because people run Playwright from their IDE, and it doesn't require the same version as Kibana does. This env variable allows us to skip the version validation. (A warning isn't an option because some processes want to fully control stdio).